packages/failurechecker/src/failurechecker.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// define types for a few modules used by probot that do not have their
+// own definitions published. Before taking this step, folks should first
+// check whether type bindings are already published.
+
+import { Application, Context } from 'probot';
+import * as util from 'util';
+
+const CONFIGURATION_FILE_PATH = 'failurechecker.yml';
+
+interface Configuration {
+  randomBoolean: boolean;
+}
+
+export = (app: Application) => {
+  app.on(
+    ['issues.opened', 'pull_request.opened', 'schedule.repository'],
+    async context => {
+      const config = (await context.config(
+        CONFIGURATION_FILE_PATH,
+        {}
+      )) as Configuration;
+
+      app.log("it's alive!");
```

Comment:
do we need it? :)

Reply:
This will be more exciting if the cron is working :stuck_out_tongue: